ccswbs/gus
# Add production build via Azure Pipeline

This has the capacity to run our production build. The pipeline currently takes 40 minutes at 1 am (to reset the cache) and 5 minutes at other times.

# Summary of changes
- Change gatsby build script to remove --prefix-paths option
- Adds schedule to run:
   - Hourly between 6 am and 6pm
   - Daily at 1 am EST to clear the Gatsby build cache
   - Daily at 2 am EST to purge the Azure CDN cache
- Create an archive of the gatsby build

## Frontend
None

## Backend
None

# Test Plan
- Confirm the build pipeline runs on Azure Devops (Select Pipelines and find the ccwbs_gus build)
- Confirm the archive is received by the Release pipeline (Select Releases and find ccs-wds-gus-deployment)
- Confirm the site is pushed to https://previewchugendpoint.azureedge.net
